tensorflow/core/protobuf/rewriter_config.proto

```diff
@@ -6,6 +6,9 @@ option java_outer_classname = "RewriterConfigProtos";
 option java_multiple_files = true;
 option java_package = "org.tensorflow.framework";
 
+// this might be a little bit overkill
```

Comment:
That's fine with me. It's difficult to change these APIs, so it's worth future proofing them a bit.